### PR TITLE
[EN] OWC 2016

### PR DESCRIPTION
**Pull request by MegaApplePi**
that took forever

### CODE REVIEW

File: wiki/Tournaments/OWC/2016/en.md [hunk 1]
```diff
@@ -0,0 +1,612 @@
...
+| Finals - Week 1    | 10-11 Dec 2016           |
+| Finals - Week 2    | 17-18 Dec 2016           |
+
+## Prizes
+
+We plan to distribute exclusive and unique merchandise to all our podium winners in different tiers. The items are yet to be announced.
+
+| Placing                                              | Prize(s)                                                                         |
+|------------------------------------------------------|:---------------------------------------------------------------------------------|
+| ![Gold Crown](/wiki/shared/GCrown.png "1st place")   | exclusive and unique merchandise, profile badge, "osu!taiko Champion" user title |
```
Comment (oralekin) [inline wiki/Tournaments/OWC/2016/en.md]: not taiko.

File: wiki/Tournaments/OWC/2016/en.md [hunk 2]
```diff
@@ -0,0 +1,612 @@
...
+   * This competition is planned as a 4 versus 4 setup.
+2. Map scoring is based on Score V2.
+3. The maps for each round will be announced by the tournament management in advance on the weekend before the actual matches take place. Only these will be used during the respective matches.
+   * There will be a NoMod, Hidden, HardRock, DoubleTime and FreeMod bracket.
+   * One map will be given as a tiebreaker map. This map will only be played in case of a tie.
+4. Match schedules will be settled by the Tournament Management. Specific regulations can be found at the bottom.
+5. If no staff or referee is available, the match will be postponed.
+6. Failed players' scores do not get added to the team score.
+   * Reviving and surviving during a map gets considered as passing it.
+7. Use of the [Visual Settings](/wiki/Game_Modifiers#Visual_Settings "Game Modifiers") options are allowed.
```
Comment (TPGPL) [inline wiki/Tournaments/OWC/2016/en.md]: Section link?

File: wiki/Tournaments/OWC/2016/en.md [hunk 3]
```diff
@@ -0,0 +1,612 @@
...
+
+### Tournament Registration
+
+1. Every user interested in joining their country's team signs up individually on the specifically created sign-up webpage.
+   - Tournament Management will create a list of potential candidates for a country's team.
+   - Tournament Management declares one candidate to the captain of the country's team, albeit temporarily.
+   - The declared captain can form their team from the candidate list of their country.
+2. To ensure valid and serious registrations, every registered user will be checked by the Tournament Management.
+   - Every registered user will be assigned to their respective country's candidate list.
+   - To be successfully accepted on the list, you have to ensure that your global osu! performace ranking is above #5000.
```
Comment (TPGPL) [inline wiki/Tournaments/OWC/2016/en.md]: Escape the hash

Comment (MegaApplePi) [inline wiki/Tournaments/OWC/2016/en.md]: no need

**Pull request merged**